site/src/App/routes/examples/basic-form/basic-form.tsx

```diff
@@ -0,0 +1,425 @@
+              <Heading level="3">Add role</Heading>
+              <Autosuggest
+                label="Job title"
+                suggestions={[
+                  { text: 'Developer' },
+                  { text: 'Designer' },
+                  { text: 'Product Manager' },
+                ]}
+              />
+              <TextField label="Company name" />
+              <MonthPicker label="Started" />
+              <Textarea label="Description" secondaryLabel="recommended" />
+            </Stack>
+          </Card>
+        </Step>
+
+        <Step
+          detail={
+            <Text component="p">
+              While we&rsquo;re at it, we can also add a `description` to
+              provider more context to the user—prompting them with what sort of
+              information they should provide.
+            </Text>
+          }
+        >
+          <Card>
+            <Stack space="large">
+              <Heading level="3">Add role</Heading>
+              <Autosuggest
+                label="Job title"
+                suggestions={[
+                  { text: 'Developer' },
+                  { text: 'Designer' },
+                  { text: 'Product Manager' },
+                ]}
+              />
+              <TextField label="Company name" />
+              <MonthPicker label="Started" />
+              <Textarea
+                label="Description"
+                secondaryLabel="recommended"
+                description="Summarise your responsibilities, skills and achievements."
+              />
+            </Stack>
+          </Card>
+        </Step>
+
+        <Step
+          heading="6. Adding the button"
+          detail={
+            <Text>
+              Now that we have all of our form fields in the card, we can now
+              add a <TextLink href="/components/Button">Button</TextLink> to
+              submit the form.
+            </Text>
+          }
+        >
+          <Card>
+            <Stack space="large">
+              <Heading level="3">Add role</Heading>
+              <Autosuggest
+                label="Job title"
+                suggestions={[
+                  { text: 'Developer' },
+                  { text: 'Designer' },
+                  { text: 'Product Manager' },
+                ]}
+              />
+              <TextField label="Company name" />
+              <MonthPicker label="Started" />
+              <Textarea
+                label="Description"
+                secondaryLabel="recommended"
+                description="Summarise your responsibilities, skills and achievements."
+              />
+              <Button>Save</Button>
+            </Stack>
+          </Card>
+        </Step>
+
+        <Step
+          detail={
+            <Stack space="large">
+              <Text component="p">
+                You&rsquo;ll notice the button is the full width of the card.
+                From a mobile perspective this is fine, but on a larger device
```

Comment:
```suggestion
                This is fine from a mobile perspective, but on larger devices
```

Reply:
"This is fine from a mobile perspective, but less that ideal on a larger device."?

Reply:
Even better 👍